plugins/plugin-chart-echarts/src/Graph/transformProps.ts

```diff
@@ -0,0 +1,191 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import {
+  CategoricalColorNamespace,
+  ChartProps,
+  getMetricLabel,
+  DataRecord,
+} from '@superset-ui/core';
+import { EChartsOption, GraphSeriesOption } from 'echarts';
+import { GraphNodeItemOption } from 'echarts/types/src/chart/graph/GraphSeries';
+import { EchartsGraphFormData, DEFAULT_FORM_DATA as DEFAULT_GRAPH_FORM_DATA } from './types';
+import { GraphConstants, tooltipConfig, normalizationLimits } from './constants';
+import { EchartsProps } from '../types';
+import { getChartPadding, getLegendProps } from '../utils/series';
+
+/* eslint-disable no-param-reassign */
+function setLabelVisibility(nodes: GraphNodeItemOption[], showSymbolThreshold: number): void {
+  if (showSymbolThreshold > 0) {
+    nodes.forEach(function (node) {
+      node.label = {
+        show: node.value! > showSymbolThreshold,
+      };
+    });
+  }
+}
+
+/* eslint-disable no-param-reassign */
+function setNormalizedSymbolSize(nodes: GraphNodeItemOption[]): void {
+  let minValue: any = Number.MAX_VALUE;
+  let maxValue: any = Number.MIN_VALUE;
```

Comment:
```suggestion
  let minValue = Number.MAX_VALUE;
  let maxValue = Number.MIN_VALUE;
```

I think you can use the inferred `number` type for these two variables. Let's try not to make the typing less strict.